File: app/src/main/java/com/iita/akilimo/entities/MaizePerformance.java
package com.iita.akilimo.entities;

import io.objectbox.annotation.Entity;
import io.objectbox.annotation.Id;
import lombok.Data;

@Data
@Entity
public class MaizePerformance {

    @Id
    long id;

    private int performanceRadioIndex;
    private String maizePerformance;
    private String performanceValue;
}

File: app/src/main/java/com/iita/akilimo/entities/MaizePerformance.kt
package com.iita.akilimo.entities

import com.j256.ormlite.field.DatabaseField
import com.j256.ormlite.table.DatabaseTable

@DatabaseTable
open class MaizePerformance {

    @DatabaseField(columnName = "id", generatedId = true)
    var id: Int? = null
    var performanceRadioIndex = 0
    var maizePerformance: String? = null
    var performanceValue: String? = null
}